account_payment_return/README.rst

```diff
@@ -16,22 +16,22 @@ full returned payment record.
 Usage
 =====
 
-Go to Invoicing > Sales > Customer Payment Returns, and create a new
-record, register on each line a paid (reconciled) receivable journal item,
-and input the amount that is going to be returned.
+1. Go to Invoicing > Sales > Customer Payment Returns, and create a new
```

Comment:
You don't need to put manually numbers. Putting `#. `, numbers are auto-assigned on rendering.